tests/specifications/general_test.py

```diff
@@ -59,15 +59,11 @@ def test_check_036():
   status, _ = list(check(sanitary_font))[-1]
   assert status == PASS
 
-  bogus_font = os.path.join("data", "test", "cabinvfbeta", "CabinVFBeta.ttf")
```

Comment:
I changed this because "CabinVFBeta.ttf" apparently passes ots-sanitize v7.1.8 without errors. Not sure why it was failing before (at least that was the previous test was asserting).